master/reference/networking-calico/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: networking-calico
+---
+
+networking-calico is the Calico sub-project that provides Calico connectivity
+and security in an OpenStack/Neutron cloud.  It provides the extra pieces that
+are needed to integrate Calico into OpenStack, namely a Neutron server driver
+or plugin, and a modified DHCP agent.
+
+Calico (http://www.projectcalico.org/) uses IP routing to provide connectivity,
```

Comment:
This isn't networking-calico specific stuff.  I don't think we should be doing the general Calico pitch here.